README.md

```diff
@@ -1545,6 +1545,25 @@ openstackbackuprestore   restore    openstackbackupsave-1641928378   Restored
 
 At this point, all resources contained with the chosen `OpenStackBackup` should be restored and fully provisioned.
 
+Exported stack data config map
+--------------------------------------------------------
+
+The OSP Director Operator automatically creates a ConfigMap after each OSDeploy resource finishes executing. This ConfigMap is named after the OSDeploy resource name and prefixed with tripleo-exports-. For example tripleo-exports-default would be the name of the ConfigMap for the 'default' OSDeploy resource. Each ConfigMap contains 2 YAML files:
+
+| Filename    | Description | TripleO Command Equivalent |
+| ----------- | ----------- | -------------------------- |
+| ctlplane-export.yaml | Used with multiple stacks for DCN | overcloud export |
+| ctlplane-export-filtered.yaml | Used for multiple stacks with Cells | overcloud cell export |
```

Comment:
maybe we want to be more specific that this is used for cell stacks where the cell controllers get installed. if you split out again cell computes from a cell controller stack you'd use the ctlplane-export.yaml from the cell controller stack.

Reply:
Sure, I added a note to indicate the cell stacks as "Controllers"